cmd/kubeadm/app/cmd/init_test.go

```diff
@@ -220,6 +221,7 @@ func TestNewInitData(t *testing.T) {
 	}
 }
 
+// Returns a function that checks if the provided initData contains the expected preflight errors.
```

Comment:
comments normally start with the func name. e.g. `expectedInitIgnorePreflightErrors` ...